### ISSUE TITLE
Links in frame comments not legible (WCAG AA)

### ISSUE THREAD

**Issue by andrewnicols** [filp/whoops#759]
When adding a link to the frame comment, it is rendered as the browsers standard link colour. In Chrome this is `rgb(0, 0, 238)` or `#0000ee`.

This is rendered on a background which is `rgba(255, 255, 255, .05)` over a background with colour `#303030`. This basically gives it a colour of `#3b3b3b`.

This combination has a [contrast ratio of 1.19:1](https://webaim.org/resources/contrastchecker/?fcolor=0000EE&bcolor=3B3B3B) and is, unfortunately, almost impossible to read.

The hover colour of `#4BB1B1` is [also a fail at 4.38:1](https://webaim.org/resources/contrastchecker/?fcolor=4BB1B1&bcolor=3B3B3B).

I'd suggest:
* remove the hover change
* add an underline to links
* apply a colour such as `#c6c6c6` which has a contrast ratio of 5.81:1 (WCAG AA pass).
